bciers/apps/compliance/src/app/components/compliance-summaries/ComplianceSummariesPage.tsx

```diff
@@ -14,28 +17,48 @@ export default async function ComplianceSummariesPage({
   const { reporting_year: reportingYear } = await getReportingYear();
   const initialData = await fetchComplianceSummariesPageData(searchParams);
   const frontEndRole = await getSessionRole();
+  const currentDate = new Date(); // Current date
+  const dueDate = new Date(reportingYear + 1, 10, 30, 23, 59, 59); // November 30 of the reporting year + 1
```

Comment:
I ~think we need a timezone on this or else it will be just before midnight on November 30 in whatever [timezone the user's browser](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date#:~:text=The%20local%20timezone%20is%20not%20stored%20in%20the%20date%20object%2C%20but%20is%20determined%20by%20the%20host%20environment%20(user%27s%20device).) is in.